packages/tokens/src/viewport.yaml

```diff
@@ -1,9 +1,9 @@
 viewport:
   mobile:
-    value: 0
+    value: '{ size.0.value }'
   tablet:
     value: 740px
   desktop:
     value: 992px
```

Comment:
These are a little non-standard... with the new primitive scale based on our 4px scaling, we could use 720 for tablet and 960 for desktop (or 1120). But we can also keep these as is if we'd rather not mess with the responsive sizes.

Reply:
or probably better idea to instead leave these as hard coded pixel values for now and we can look at breakpoints separately

Reply:
No strong preference either way. wdyt @apucacao since you worked on breakpoints a bit for the product?

Reply:
i think leaving them as-is if it doesn't impede our progress with the new design system stuff here makes sense to me.

Reply:
Cool let's leave as defined for now :)